java-manta-client-unshaded/src/main/java/com/joyent/manta/config/ConfigContextMBean.java

```diff
@@ -98,6 +98,10 @@ public ConfigContextMBean(final ConfigContext config) {
                 // MapConfigContext.MANTA_PRIVATE_KEY_CONTENT_KEY omitted
                 // MapConfigContext.MANTA_PASSWORD_KEY omitted
                 // MapConfigContext.MANTA_NO_AUTH_KEY omitted
+                new MBeanAttributeInfo(MapConfigContext.MANTA_TLS_INSECURE_KEY,
+                        Boolean.class.getName(),
+                        "Flag that indicates of TLS certificate verification is disabled",
```

Comment:
Suggestion: Description could be refactored `Flag that indicates of `-> `Flag indicating TLS certificate verification is disabled`